griptape/drivers/vector/amazon_opensearch_vector_store_driver.py

```diff
@@ -0,0 +1,36 @@
+import boto3
+from requests_aws4auth import AWS4Auth
+from attr import define, field, Factory
+from typing import Optional, Union, Tuple
+from griptape.drivers import OpenSearchVectorStoreDriver
+from opensearchpy import OpenSearch, RequestsHttpConnection
+
+
+@define
+class AmazonOpenSearchVectorStoreDriver(OpenSearchVectorStoreDriver):
+    session: boto3.session.Session = field(kw_only=True)
+    region_name: str = field(kw_only=True)
+    host: str = field(kw_only=True)
+    port: int = field(default=443, kw_only=True)
+    use_ssl: bool = field(default=True, kw_only=True)
+    verify_certs: bool = field(default=True, kw_only=True)
+    index_name: str = field(kw_only=True)
```

Comment:
We don't need those since they are going to be inherited from `OpenSearchVectorStoreDriver`.